src/styles/product.scss

```diff
@@ -85,3 +85,8 @@
     }
   }
 }
+
+.product:hover {
+  transform: scale(1.2);
+  transition-duration: 0.5s;
```

Comment:
This transition should be added for product card

Reply:
> This transition should be added for product card

this comment not fixed.
add transition for .product selector. it will make smooth styles changing in two ways.